docs/displaying-ads.mdx

```diff
@@ -107,7 +107,7 @@ function App() {
   const [loaded, setLoaded] = useState(false);
 
   useEffect(() => {
-    const unsubscribe = interstitial.addAdEventListener(AdEventType.LOADED => {
+    const unsubscribe = interstitial.addAdEventListener(AdEventType.LOADED, => {
```

Comment:
```suggestion
    const unsubscribe = interstitial.addAdEventListener(AdEventType.LOADED, () => {
```

Reply:
Ah I see! The example app does not actually show any usage of `addAdEventListener` (it's still using the multi-event listener style), so when I consulted it I did not have a good reference. So I thought the event type was the param, not that it was a list of params and the arrow method needed an empty param list :-)